raiden/tests/scenarios/join-network-scenario-J11.yaml

```diff
@@ -0,0 +1,41 @@
+settings:
+  gas_price: "fast"
+
+token:
+
+  address: "0xfE76bc6FeF416Bf8270822104da293cDAEC20E1e"
+
+nodes:
+  commands:
+    stop: "pkill -f '127.0.0.1:50{:02d}'"
+    kill: "pkill -9 -f '127.0.0.1:50{:02d}'"
+    start: "raiden --config-file raiden_{0:02d}.toml --api-address 127.0.0.1:50{0:02d}"
+  range:
+    first: 1
+    last: 4
+    template: "localhost:50{:02d}"
+
+scenario:
+  serial:
+    tasks:
+      - parallel:
+          name: "Setting up a network"
+          tasks:
+            - open_channel: {from: 0, to: 1, total_deposit: 10, expected_http_status: 201}
+            - open_channel: {from: 0, to: 2, total_deposit: 10, expected_http_status: 201}
+            - open_channel: {from: 1, to: 2, total_deposit: 10, expected_http_status: 201}
+
+      - serial:
+          name: "Checking the network"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 10, balance: 10, state: "opened"}
+            - assert: {from: 0, to: 2, total_deposit: 10, balance: 10, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 10, balance: 10, state: "opened"}
+
+      - serial:
+          name: "Node Nr. 4 joins a network with 100 funds and joinable_funds_target set to 1"
+          tasks:
+            - join_network: {from: 3, funds: 100, joinable_funds_target: 1, expected_http_status: 204}
+            - assert: {from: 3, to: 0, total_deposit: 0, balance: 0, state: "opened"}
```

Comment:
Shouldn't this be 33 each?

Reply:
No `joinable_funds_target` is the part of `funds` that is being reserved for depositing into future channels, when new participants join the network

Reply:
Ah, thanks for the explanation.